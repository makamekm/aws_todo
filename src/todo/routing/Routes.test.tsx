import {
    getRoutes,
} from "./";

describe("Todo", () => {
    describe("Routing", () => {

        it("should match snapshot", async () => {
            expect(getRoutes()).toMatchSnapshot();
        });
    });
});
