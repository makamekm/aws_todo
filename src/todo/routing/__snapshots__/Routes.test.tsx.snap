// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todo Routing should match snapshot 1`] = `
Array [
  Object {
    "component": [Function],
    "exact": true,
    "path": "/",
  },
  Object {
    "component": [Function],
    "path": "/index",
  },
]
`;
